### PR TITLE
fix(curriculum): remove jQuery from Build a Recipe Page lab

### PR DESCRIPTION
**Pull request by royjohnlee**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #[56489](https://github.com/freeCodeCamp/freeCodeCamp/issues/56489)
Removed all jQuery tests.
Used specific assert methods where applicable.
Corrected hint to match the text.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md [hunk 1]
```diff
@@ -77,19 +77,19 @@ assert(code.match(/<\s*html[\s\S]*>[\s\S]*<\s*head\s*>[\s\S]*<\/\s*head\s*>[\s\S
...
 ```js
-assert.lengthOf(document.querySelectorAll('h1'), 1)
+assert.lengthOf(document.querySelectorAll('h1'), 1);
 ```
 
 You should have a `p` element below your `h1` element with text describing your recipe.
 
 ```js
-assert($('p')[0].previousElementSibling.tagName === 'H1');
+assert.isTrue(document.querySelector('p')?.previousElementSibling.tagName === 'H1');
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md:92]:
```suggestion
assert.strictEqual(document.querySelector('p')?.previousElementSibling.tagName, 'H1');
```

Please update the others as well.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md (hunk 1 above)
Comment (royjohnlee) [reply curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md]: understood!

File: curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ assert.equal(document.querySelectorAll('h2')[0]?.innerText, 'Ingredients');
 You should have an unordered list element below your first `h2` element.
 
 ```js
-assert($('ul')[0].previousElementSibling.tagName === 'H2');
+assert.isTrue(document.querySelector('ul')?.previousElementSibling.tagName === 'H2');
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md:104]:
```suggestion
assert.strictEqual(document.querySelector('ul')?.previousElementSibling.tagName, 'H2');
```
These as well.

File: curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ assert.equal(document.querySelectorAll('h2')[1]?.innerText, 'Instructions');
 You should have an ordered list element below your second `h2` element.
 
 ```js
-assert($('ol')[0].previousElementSibling.tagName === 'H2');
+assert.isTrue(document.querySelector('ol')?.previousElementSibling.tagName === 'H2');
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md:124]:
```suggestion
assert.strictEqual(document.querySelector('ol')?.previousElementSibling.tagName, 'H2');
```

**Review by naomi-lgbt — APPROVED** [review]

**Review by gikf — APPROVED** [review]

**Pull request merged**